### DISCUSSION TITLE
"Keyword" is not providing better results with live streaming audio which has pharmaceutical product names

### DISCUSSION THREAD

**Discussion by vanithaVP** [deepgram/community#139] [General help] [answered]
I am trying to transcribe live streaming audio which has pharmaceutical product names in the conversation. I am using nova model with keywords which has all expected product names will be mentioned in the conversation. The transcribed text accuracy is inconsistent.

Audio: I discuss Rituxan Imbruvica Synribo Bosulif Lunsumio
Transcript1: i discuss Rituxan Imbruvica Synribo Lunsumio _(missed Bosulif)_
Transcript 2: i discuss Rituxan Imbruvica and rivo Bosulif Lunsumio _(Synribo misspelled)_

I read custom model is not supported in live streaming. Can you help me on fine tuning deepgram for my usecase?
Thanks

**Comment by shirgoldbird** [comment, marked as answer]
@vanithaVP Custom models are supported for live streaming, and might be a good fit for your use case (feel free to reach out to our [sales team](https://deepgram.com/contact-us/) if you'd like more information!). However, you can also use keywords to fine-tune performance. In addition to your keywords, please add the parameter `keyword_boost=standard` to your streaming request to get them to work with Nova. 

Apologies that this is poorly documented at the moment; we're about to roll out a change so you'll no longer need to set the `keyword_boost` parameter.

**Reply by vanithaVP** [reply]
Thanks, @shirgoldbird. keyword_boost=standard works better for my use case. Do we have any limitation of using keywords? I want to add 400+ keywords for my case. Is this possible?

**Reply by shirgoldbird** [reply]
@vanithaVP As keywords are passed in via URL, you're constrained by URL character limits (2048 characters), and the keywords feature is also not designed to support so many special terms. Beyond 50 keywords or so, that many keywords suggests you may want to talk to our sales team about custom model training.
